app/HutchAgent/hutchagent/db_manager.py

```diff
@@ -87,7 +87,9 @@ def __init__(
             port=port,
             database=database,
         )
-        self.engine = create_engine(url=url)
+        self.engine = create_engine(
+            url=url, connect_args={"options": "-csearch_path={}".format("omop")}
+        )
```

Comment:
won't this only work if connecting to _our_ omop db?

Reply:
yes that's true, I could set it as env variable and make it optional?

Reply:
`dict`s aren't supported in .env files

Reply:
Hang on, I suppose the bit where it says "omop" can be configured. Then I suppose it could be set to "public", or whatever to target the right schema in other places.

Reply:
Yeah that's the part I meant, should I keep it like this in that case?

Reply:
Yeah. Just put the env variable where omop should be. Sorry for not understanding the first time.